assets/dev/js/editor/regions/panel/pages/elements/elements.js

```diff
@@ -62,16 +62,7 @@ PanelElementsLayoutView = Marionette.LayoutView.extend( {
 	},
 
 	initElementsCollection: function() {
-		var elementsCollection = new PanelElementsElementsCollection(),
-			sectionConfig = elementor.config.elements.section;
-
-		elementsCollection.add( {
-			title: __( 'Inner Section', 'elementor' ),
-			elType: 'section',
-			categories: [ 'basic' ],
-			keywords: [ 'row', 'columns', 'nested' ],
-			icon: sectionConfig.icon,
-		} );
+		var elementsCollection = new PanelElementsElementsCollection();
```

Comment:
I know it's not your code, but anyway

```suggestion
		const elementsCollection = new PanelElementsElementsCollection();
```